AWSTestResources/AWSTestResources.h

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
The implementation is still TODO for this right?

Reply:
Correct